pytorch_lightning/utilities/imports.py

```diff
@@ -181,6 +181,11 @@ def __repr__(self) -> str:
 else:
     _HPU_AVAILABLE = False
 
+if _TORCH_GREATER_EQUAL_1_12:
+    _MPS_AVAILABLE = torch.backends.mps.is_available()
+else:
+    _MPS_AVAILABLE = False
```

Comment:
Can we put this in the MPS accelerator file directly? https://github.com/PyTorchLightning/pytorch-lightning/issues/11826